requirements-dev.txt

```diff
@@ -0,0 +1,6 @@
+flake8
+cookiecutter
+pytest
+recommonmark
+Sphinx
+sphinx-markdown-tables
```

Comment:
I think the intention is for `requirements-dev.txt` to be used by developers of `govcookiecutter`, rather than users. By users, I mean someone who just wants to build a repository from this template, rather than make edits/improvements/bugfixes (developer).

In this case, `requirements.txt` needs to have `cookiecutter`, so a user can run it. Developers also need `cookiecutter`, if they'd like to test build it themselves. When developing, Developers should also ensure all tests pass (hence the `pytest` requirement), and that linting (`flake8`) has been performed.